### PR TITLE
working on ens util, available ENS func only available to user

### PR DESCRIPTION
**Pull request by 0xSero**
## Description

Adding a Util to query for ENS

## Related Issue

[ENS](https://www.notion.so/Create-Query-ENS-util-b8b19f19fc3c47adb2b48245a70a4108)

## Motivation and Context

Display ENS

## How Has This Been Tested?

No testing, not ready

### CODE REVIEW

File: src/utils/format.ts [hunk 1]
```diff
@@ -130,6 +130,13 @@ export const formatAddress = (address: string) => {
   return address.substring(0, 6) + '...' + address.substring(address.length - 4, address.length);
 };
 
+export const getENSname = (address: string) => {
+  //TO DO: need to set up a provider here.
```
Comment (kibagateaux) [inline src/utils/format.ts]: pass provider in as an arg. We'll always have access to it from actions

Comment (0xSero) [reply src/utils/format.ts]: What actions are you referencing, I just checked all store Actions and found nothing related to providers

**I suppose we need to create a new selector for this**

**Comment by 0xSero** [comment]
This needs some additional work and will be tackled after other PRs are merged

File: src/utils/format.ts (hunk 1 above)
Comment (kibagateaux) [reply src/utils/format.ts]: `wallet.provider`

File: src/utils/format.ts [hunk 2]
```diff
@@ -130,6 +131,22 @@ export const formatAddress = (address: string) => {
   return address.substring(0, 6) + '...' + address.substring(address.length - 4, address.length);
 };
 
+export const getENSname = async (address: string) => {
```
Comment (kibagateaux) [inline src/utils/format.ts:134]: why is same code in two places?

**Comment by 0xSero** [comment]
This is being resolved in PR:

https://github.com/debtdao/marketplace-app/pull/68

**Pull request closed without merging**